target_postgres/tests/test_target_postgres.py

```diff
@@ -406,11 +443,111 @@ def test_duplicate_records(postgres_target):
     verify_data(postgres_target, "test_duplicate_records", 2, "id", row)
 
 
-def test_array_data(postgres_target):
-    file_name = "array_data.singer"
+def test_array_boolean(postgres_target):
+    file_name = "array_boolean.singer"
+    singer_file_to_target(file_name, postgres_target)
+    row = {"id": 1, "value": [True, False]}
+    verify_data(postgres_target, "array_boolean", 3, "id", row)
+    verify_schema(
+        postgres_target,
+        "array_boolean",
+        check_columns={
+            "id": {"type": BIGINT},
+            "value": {"type": ARRAY},
+        },
+    )
+
+
+@pytest.mark.skip("Needs pgvector support")
+def test_array_float_vector(postgres_target):
+    file_name = "array_float_vector.singer"
+    singer_file_to_target(file_name, postgres_target)
+    row = {
+        "id": 1,
+        "value": [Decimal("1.0"), Decimal("2.0"), Decimal("1.0"), Decimal("1.0")],
+    }
+    verify_data(postgres_target, "array_float_vector", 3, "id", row)
+    verify_schema(
+        postgres_target,
+        "array_float_vector",
+        check_columns={
+            "id": {"type": BIGINT},
+            "value": {"type": ARRAY},
+        },
+    )
```

Comment:
### About

This one (currently skipped) would need pgvector support enabled, and corresponding support by the type system.

Let me know if you think it is a good idea to explore this feature within `target-postgres`, or if you think it should be implemented on behalf a different target, like `target-pgvector`.

### References
- https://github.com/meltano/sdk/issues/2102
- https://github.com/crate-workbench/meltano-target-cratedb/issues/5

Reply:
I've deferred this snippet to a different patch.

- GH-251